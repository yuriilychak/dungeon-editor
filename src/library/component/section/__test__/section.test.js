import React from "react";
import {createMount} from "@material-ui/core/test-utils";

import Section from "../section";

describe("Section test", () => {
    it("default snapshot", () => {
        const wrapper = createMount()(
            <Section
                id={0}
                icon="test"
                files={[]}
                addDirectoryText="test"
                addElementText="test"
                emptyText="test"
                deleteText="test"
                renameText="test"
                titleText="test"
                onAddFile={jest.fn()}
                onAddDirectory={jest.fn()}
                onUpdateTree={jest.fn()}
                onRemoveFile={jest.fn()}
                onRenameFile={jest.fn()}
                onSelectFile={jest.fn()}
            />
        );

        expect(wrapper.html()).toMatchSnapshot();
    });
});
