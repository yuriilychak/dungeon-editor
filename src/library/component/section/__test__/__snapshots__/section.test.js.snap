// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section test default snapshot 1`] = `"<div class=\\"MuiPaper-root MuiExpansionPanel-root MuiExpansionPanel-rounded MuiPaper-elevation1 MuiPaper-rounded\\"><button><div class=\\"MuiExpansionPanelSummary-content\\"><img src=\\"undefined/static/icon/test_root.svg\\" style=\\"width:20px;height:20px\\" class=\\"icon-root\\" alt=\\"empty Icon\\" draggable=\\"false\\"/><span class=\\"expansion-panel-title\\">test</span><div><span>Tooltip mock</span><div><button><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z\\"></path><path fill=\\"none\\" d=\\"M0 0h24v24H0z\\"></path></svg></span></button></div></div><div><span>Tooltip mock</span><div><button><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path fill=\\"none\\" d=\\"M0 0h24v24H0V0z\\"></path><path d=\\"M20 6h-8l-2-2H4c-1.11 0-1.99.89-1.99 2L2 18c0 1.11.89 2 2 2h16c1.11 0 2-.89 2-2V8c0-1.11-.89-2-2-2zm-1 8h-3v3h-2v-3h-3v-2h3V9h2v3h3v2z\\"></path></svg></span></button></div></div></div><button><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z\\"></path><path fill=\\"none\\" d=\\"M0 0h24v24H0z\\"></path></svg></span></button></button><div class=\\"MuiCollapse-container MuiCollapse-hidden\\" style=\\"min-height:0px\\"><div class=\\"MuiCollapse-wrapper\\"><div class=\\"MuiCollapse-wrapperInner\\"><div aria-labelledby=\\"0\\" id=\\"0\\" role=\\"region\\"><div class=\\"MuiExpansionPanelDetails-root\\"><span class=\\"section-body-empty\\">test</span></div></div></div></div></div></div>"`;
